src/app/Quiz.js

```diff
@@ -0,0 +1,41 @@
+import { shuffle } from './shuffle';
+import { findImageUrl } from './findImageUrl'
+export class Quiz {
+  constructor() {
+    this.people = [];
+    this.vehicles = [];
+    this.starships = [];
+  }
+  setData(query, data) {
+    switch (query) {
+      case 'people':
+        this.people.push(data);
+        break;
+      case 'vehicles':
+        this.vehicles.push(data);
+        break;
+      case 'starships':
+        this.starships.push(data);
+        break;
+    }
+  }
+  getRandomQuizObject(gameMode) {
+    let indices = [];
+    let name, imgIndex;
+    while (indices.length < 4) {
+      let randomIndex = Math.floor(Math.random() * 9) + 1; //9 is hardcoded becasue this is exactly how many we get from swapi
```

Comment:
Tak na przyszłość: zazwyczaj jak musimy gdzieś zapisać taką stałą lepiej sobie stworzyć na to zmienią i nazwać ją odpowiednio zamiast pisać komentarz, np.: `const numberOfPeoplesInAPIResponse = 10`